WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/bloggingprompts/BloggingPromptCardViewHolder.kt

```diff
@@ -31,6 +34,7 @@ class BloggingPromptCardViewHolder(
 ) : CardViewHolder<MySiteBloggingPromptCardBinding>(
     parent.viewBinding(MySiteBloggingPromptCardBinding::inflate)
 ) {
+    @SuppressLint("ClickableViewAccessibility")
```

Comment:
Do we still need this suppress annotation? I've removed it locally and didn't find any warnings.